meta-client/src/main/java/org/apache/helix/metaclient/impl/zk/factory/ZkMetaClientFactory.java

```diff
@@ -36,4 +38,17 @@ public MetaClientInterface getMetaClient(MetaClientConfig config) {
     }
     throw new IllegalArgumentException("Invalid MetaClientConfig type.");
   }
+
+  @Override
+  public MetaClientCacheInterface getMetaClientCache(MetaClientConfig config, String key, Boolean cacheData,
+                                                     Boolean cacheChildren, Boolean lazyCaching) {
```

Comment:
Can we group the later 4 vars into a cache config?

Reply:
Are you saying to have a cache config and a zk cache config too?

Reply:
I was thinking having a cachConfig that has the folling 4 vars.
String key, Boolean cacheData, Boolean cacheChildren, Boolean lazyCaching